frontend/src/components/permit/PermitModal.vue

```diff
@@ -221,13 +222,18 @@ async function onSubmit(data: PermitForm, { resetForm }) {
         <InputText
           class="col-12 lg:col-6"
           name="issuedPermitId"
-          label="Permit ID"
+          label="Issued Permit ID"
         />
         <Calendar
           class="col-12 lg:col-6"
           name="adjudicationDate"
           label="Adjudication date"
         />
+        <Calendar
+          class="col-12 lg:col-6"
+          name="lastVerifiedDate"
+          label="Last verified date"
+        />
```

Comment:
Submitting a future date fails silently on the frontend but throws error on the backend, there's no notification to the user exactly what happened. Consider adding validation that triggers an error on the frontend on future dates, or disallow future dates from being entered, or both.